### ISSUE TITLE
Version number link and text

### ISSUE THREAD

**Issue by yiwenxx** [pulibrary/tigerdata-app#1230]
### **Issue**

1. Version number link should be white in color instead of blue, failing color contrast accessibility.
2. Missing divider between version and last updated information.
3. "Last Updated" should be capitalized.
4. No spacing between version information and divider line.
<br>
<img width="1511" alt="Image" src="https://github.com/user-attachments/assets/56fd3098-d417-4843-a80b-9de3f7f0986b" />

<br>

Hover effect:
![Image](https://github.com/user-attachments/assets/e838ae3b-2bef-4b53-a36d-d2f7adac836c)

<img width="1136" alt="Image" src="https://github.com/user-attachments/assets/c56157fa-c53b-4f25-8e2a-b9c60ccf535c" />
<br>

Link to Figma: https://www.figma.com/design/rgzEYTpnGl6l68moRSVwYg/Phase-1-UI-Details?node-id=1750-14955&t=I3GXx7fYln7e6HP5-4

## Acceptance criteria

- [ ] In default state, version number has the same white color as the text, with underline; when hover, version number and underline change color to Medium Grey (#D0D0D0).
- [ ] Capitalized the first letter of "Last Updated".
- [ ] Add divider (|) in between the version number and the 'Last Updated', the spacing is 4px on both sides.
- [ ] Nice to have: add a 8px spacing in between version information and divider.

**Comment by kelynch** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/RDSS-61a4f1a12a399b001730f65a/issues/pulibrary/tigerdata-app/1230?planning-poker) with Zenhub @bess @JaymeeH @jrgriffiniii
